### ISSUE TITLE
License

### ISSUE THREAD

**Issue by bstoots** [VestergaardCompany/js-mixin#2]
Great update to the original mixwith project.  Any particular reason you licensed under GPL-3 when the original project was Apache-2.0 ?  I would like to use this for a project but I'm not interested in getting tangled up with GPL'ed code.  Would you consider re-licensing this under Apache-2.0?

**Comment by aedart** [comment]
Hi @bstoots 

To the best of my ability, I'm writing to you in an official capacity, as a software developer employed at [Vestergaard Company A/S](https://vestergaardcompany.com/) - and not as a private software author.
Furthermore, I am by no means an expert in software licensing. Therefore, please bare with me, should my answer contain incorrect statements about GPL-3 or Apache-2.0 licenses.

## Re-Licensing
As mentioned in the README.md, this is an adaptation of the original package. While the core functionality remains the same as the original, significant changes have been made to the actual source code.
We consider this package to be "derivative work". [Apache-2.0 licensing conditions](https://en.wikipedia.org/wiki/Apache_License#Licensing_conditions) does allow us to distribute this package under a different license than the original, due to the mentioned reason.
Our intent is to ensure that this package always remained available to ourselves and others. Therefore, we published in under GPL-3 and have no intent to change this license form.

## GPL-3 Compliance
If you are worried about using this package in commercial software, then please read the [CopyLeft](https://en.wikipedia.org/wiki/GNU_General_Public_License#Copyleft) description, as well as [compatibility with other licenses](https://en.wikipedia.org/wiki/GNU_General_Public_License#Compatibility_and_multi-licensing).

In short, this license affects this package - not the entirety of your project. We cannot claim ownership or "special usage privileges" over your project. For instance, if you are using our code for some single page application (e.g. on a website), then simple link to this package in the distributed code's comments, _should be sufficient_ to be compliant with GPL-3. I explicitly use the phrase _should be sufficient_, because I know that JavaScript typically gets transpiled, transformed, minimised, bundled... etc. The final output is therefore typically far from human-readable, and it becomes difficult to verify any breaches of license(s).

I hope that this reply answers your questions.

**Comment by bstoots** [comment]
I appreciate the quick response.  You are well within your rights to license this project as you see fit, I don't hold it against you in any way.  There may be bundling and distribution in the future of my derivative work so unfortunately I will be erring on the side of caution and avoiding js-mixin for this particular project.
